src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from "react";
+import CssBaseline from "@mui/material/CssBaseline";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as yup from "yup";
+
+import { useNavigate } from "react-router-dom";
+
+import styled from "styled-components";
+
+const Login = styled.div`
+  @media (min-width: 1200px) {
+    background-color: #c2d6d6;
+    height: 100vh;
+    width: 100vw;
+    align-items: center;
+    justify-content: center;
+    padding-top: 50px;
+  }
+
+  @media (max-width: 480px) {
+    background-color: none;
+  }
+`;
+
+const Lbody = styled.div`
+  width: 500px;
+  height: 500px;
+  background-color: white;
+  opacity: 0.8;
+  align-items: center;
+  margin: auto;
+  padding-top: 25px;
+
+  @media (max-width: 768px) {
+    margin-top: 50px;
+  }
+
+  @media (max-width: 480px) {
+    margin-top: 150px;
+  }
+`;
+
+const Lh1 = styled.h1`
+  margin-left: 30px;
+  margin-top: 20px;
+`;
+
+const ErrMsg = styled.span`
+  color: red;
+`;
+
+function LogIn() {
+    const initialValues = {
+        email: "",
+        password: "",
+    };
+    const validationSchema = yup.object().shape({
+        email: yup.string().email("Please Enter Valid Email").required("Required"),
+        password: yup.string().required("Required"),
+    });
+    const onSubmit = (values: any) => {
+        console.log(values);
+    };
+
+    const navigate = useNavigate();
+
+    const navigateRegister = () => {
+        navigate("/register");
+    }
+    return (
+        <>
+            <CssBaseline />
```

Comment:
Move this CssBaseline to the root element then you don't have to add this to every file